app/views/layouts/application.html.erb

```diff
@@ -57,6 +57,12 @@
                     <%= link_to t('layout.application.admin') , admin_path, class: "govuk-header__link" %>
                   </li>
                 <% end %>
+                <li class="govuk-header__navigation-item">
+                  <%= link_to t('layout.application.documentation'), "https://prototype-docs.cloudapps.digital/rotating-your-keys-and-certificates/#rotating-your-keys-and-certificates", class: "govuk-header__link" %>
```

Comment:
any chance the documentation URL can be pulled from the locales as with the other ones? At some point we'll have to update all of the links and it will be easier if they're all in a single place.
https://github.com/alphagov/verify-self-service/blob/2371670e8c40139a60ee3a8ecefe1f975dcf2c4b/config/locales/en.yml#L354

Reply:
Good idea :) I have amended.